features/support/time_machine_helpers.rb

```diff
@@ -48,4 +48,12 @@ def run(tasks)
       Sidekiq::Testing.drain_batches
     end
   end
+
+  def time_frozen?
+    before = Time.zone.now.to_f
+    sleep 0.1
```

Comment:
```suggestion
    sleep 0.01
```

I guess these 100 ms are just noise if test does any DB call. But in the spirit of nitpicking :innocent: 
You can ignore this.

In either case, nice solution, I remember seeing this in the past but lack of API to know if time was frozen made me ignore. I'm not sure how good it is to ever move time back but I guess some tests need.

Reply:
I changed it to 0.0001